qemu/tests/numa_memdev_options.py

```diff
@@ -123,3 +154,4 @@ def run(test, params, env):
 
     error_context.context("Check the memory in procfs!", logging.info)
     check_memory_in_procfs(test, params, vm)
+    vm.verify_kernel_crash()
```

Comment:
Would you please add vm.wait_for_login() before verify_kernel_crash() to make sure guest boot up successfully? Others look good to me, thanks.

Reply:
Or just use vm.verify_dmesg() instead ?